edison/README.md

```diff
@@ -15,10 +15,32 @@ make a decision.
 The library is using a Feature Extraction Language (FEX) as a declarative way of defining features. 
 that can be applied to text to generate features. 
 
-## Concepts
-   - Feature extractors
-   - Feature input transformers
-   - Operations
+## Maven dependency
+To use Edison as a Maven dependency please add the following lines to your `pom.xml` file:
+```
+<repositories>
+    <repository>
+        <id>CogcompSoftware</id>
+        <name>CogcompSoftware</name>
+        <url>http://cogcomp.cs.illinois.edu/m2repo/</url>
+    </repository>
+</repositories>
+
+<dependencies>
+    <dependency>
+        <groupId>edu.illinois.cs.cogcomp</groupId>
+        <artifactId>illinois-edison</artifactId>
+        <version>3.0.42</version>
+    </dependency>
+</dependencies>
+```
+
+or if you are using SBT:
+```
```

Comment:
Actually for the time being I want to keep it that way since it makes the copy-paste much easier. In the future we should find a better way to fix this (maybe always point to the latest Maven central released version)